packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -343,8 +341,8 @@ export class MgtPersonCard extends MgtTemplatedComponent {
    */
   protected sections: BasePersonCardSection[];
 
-  @internalProperty() private state: MgtPersonCardState;
-  @internalProperty() private isStateLoading: boolean;
+  @state() private state: MgtPersonCardState;
+  @state() private isStateLoading: boolean;
```

Comment:
```suggestion
  @state() private _cardState: MgtPersonCardState;
  @state() private _isStateLoading: boolean;
```

@gavinbarron I've changed these variable names and updated them in the file. `state` is imported above and might not cause any harm but averting a possibility of name collisions. Then made them `_privateVariable` syntax for uniformity. Otherwise looks good to go!

Reply:
Thanks @musale that's a really good change to make, Much appreciated!